src/components/Footer.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useMemo } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { FilterBy } from '../types/FilterBy';
+
+type Props = {
+  todos: Todo[];
+  todoFilterType: string;
+  setTodoFilterType: (todoFilterType: FilterBy) => void;
+};
+
+const Footer: React.FC<Props> = ({
+  todos,
+  todoFilterType,
+  setTodoFilterType,
+}) => {
+  const activeTodosLeft = useMemo(() => {
+    return todos.filter((todo) => !todo.completed);
+  }, [todos]);
+
+  return (
+    <>
+      {/* Hide the footer if there are no todos */}
+      <footer className="todoapp__footer">
+        <span className="todo-count">{`${activeTodosLeft.length} items left`}</span>
+
+        {/* Active filter should have a 'selected' class */}
+        <nav className="filter">
+          <a
+            href="#/"
+            className={classNames('filter__link', {
+              selected: todoFilterType === FilterBy.all,
+            })}
+            onClick={() => setTodoFilterType(FilterBy.all)}
+          >
+            All
+          </a>
+
+          <a
+            href="#/active"
+            className={classNames('filter__link', {
+              selected: todoFilterType === FilterBy.active,
+            })}
+            onClick={() => setTodoFilterType(FilterBy.active)}
+          >
+            Active
+          </a>
+
+          <a
+            href="#/completed"
+            className={classNames('filter__link', {
+              selected: todoFilterType === FilterBy.completed,
+            })}
+            onClick={() => setTodoFilterType(FilterBy.completed)}
+          >
+            Completed
+          </a>
+        </nav>
+
+        {/* don't show this button if there are no completed todos */}
```

Comment:
Pay attention to this commented line. (And remove this line from the code)
<img width="603" alt="image" src="https://user-images.githubusercontent.com/99403198/229089958-bb3a0f7f-12e8-4476-b5ce-e6eb86828587.png">